docs/node/getting-started.md

```diff
@@ -104,6 +104,10 @@ version: v2
 plugins:
   - local: protoc-gen-es
     out: gen
+    # Also generate any well-known type dependencies
+    include_wkt: true
```

Comment:
Can you give an example for why this is useful?

Reply:
Actually just realized we probably don't need that in the docs and should probably just use `include_imports`. Will update